packages/core/src/components/templates/Loader/LoaderPopup.tsx

```diff
@@ -61,6 +63,7 @@ const LoaderPopup = ({
   const {processItem, loading} = useLoaderListner(
     {
       disabled,
+      autoLeave,
```

Comment:
I'm not sure, it's useful to add it to the process, you are not using it on the provider. 
What you did in the timeout handler should be sufficient.

Reply:
I added it as process item props in order to save it as history of options related to process.